qcengine/programs/tests/standard_suite_ref.py

```diff
@@ -3270,6 +4050,53 @@
                         calc["data"]["SCS-MP2 CORRELATION ENERGY"] + calc["data"]["HF TOTAL ENERGY"]
                     )
 
+        if "MP3 CORRELATION ENERGY" in calc["data"]:
```

Comment:
It's not clear to me why there's nesting in this sequence of conditions. Can some of the nesting be done without?

Reply:
Many implementations are incomplete wrt subsidiary figures (e.g., singles energy). So unless one can collect a complete set of ref values for all r/u/rohf, conv/df/cd combinations, it's safer to do the nesting.

Reply:
Ah, thank you! This clarifies the intent for me. I'm not familiar enough with the interdependence of these figures to reason about what the nesting means here.